### PR TITLE
Support local and remote working dir config for remote execution ssh

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by ekohl** [comment]
:+1: on green tests

File: templates/plugin/remote_execution_ssh.yml.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: <%= @module_enabled %>
 :ssh_identity_key_file: <%= scope.lookupvar('::foreman_proxy::plugin::remote_execution::ssh::ssh_identity_path') %>
+:local_working_dir: <%= @local_working_dir %>
```
Comment (domcleal) [inline templates/plugin/remote_execution_ssh.yml.erb]: You need to use scope.lookupvar here, as they're not necessarily in scope on Puppet 4.

**Comment by mmoll** [comment]
@stbenjam please use scope.lookupvar, the other test failures are known and unrelated anyway. :boom:

**Comment by stbenjam** [comment]
Updated, hopefully Travis finishes before our sun goes supernova :sunny:

**Comment by mmoll** [comment]
merged as 2c7f48a, thanks @stbenjam!

**Pull request closed without merging**